client/src/app/site/services/model-request.service.ts

```diff
@@ -155,11 +144,35 @@ export class ModelRequestService {
         }
     }
 
+    private async lazyLoadSubscription(
+        { modelRequest, subscriptionName, ...config }: SubscribeToConfig,
+        availableIds: Ids
+    ): Promise<void> {
+        const childModelRequest = {
+            viewModelCtor: modelRequest.lazyLoad.ownViewModelCtor,
+            ids: availableIds.slice(0, 20),
+            fieldset: modelRequest.lazyLoad.fieldset
+        };
+        const childSubscriptionName = `${subscriptionName}_1`;
+        this.makeSubscription({
+            modelRequest: childModelRequest,
+            subscriptionName: childSubscriptionName,
+            hideWhen: timer(3000).pipe(concatMap(() => of(true)))
+        });
+        setTimeout(async () => {
+            this.makeSubscription({ modelRequest, subscriptionName, ...config });
+        }, Math.floor(Math.random() * 2000) + 2000);
+    }
+
     private setCloseFn(requestFamilyName: string, observable: Observable<boolean>): void {
         this._observableSubscriptionMap[requestFamilyName] = observable.subscribe(isTrue => {
             if (isTrue) {
                 this.closeSubscription(requestFamilyName);
             }
         });
     }
+
+    private isValidId(id: number): boolean {
+        return !Number.isNaN(id) || id > 0;
+    }
```

Comment:
I think it would be better to place this in `client/src/app/domain/definitions/key-types.ts` or `utils`.

Also:
```suggestion
    private isValidId(id: Id): boolean {
        return !Number.isNaN(id) || id > 0;
    }
```